sdk/core/core-amqp/package.json

```diff
@@ -71,6 +71,8 @@
     "rhea-promise": "^0.1.15"
   },
   "devDependencies": {
+    "@azure/identity": "^0.1.0",
+    "@azure/core-http": "^1.2.6",
```

Comment:
`@azure/core-http` will need to be in the `dependencies` section since it's required at runtime, not just development time.  `@azure/identity` should definitely stay in `devDependencies` though!

Reply:
Agreed.

On a side note, @daviwil for Preview 2, we should consider moving the `TokenCredential` related info out of `core-http` or have a copy in `core-amqp` depending on how much of a package size increase does including `core-http` causes.

Reply:
Yeah, I've been thinking that too.  `@azure/core-http` is very heavyweight when you really only just want the interface definition.

Reply:
Can we have an issue to track this?